modules/graphdb/templates/04_gdb_conf_overrides.sh.tpl

```diff
@@ -22,7 +22,7 @@ echo "#######################################"
 
 LB_DNS_RECORD=${graphdb_lb_dns_name}
 NODE_DNS_RECORD=$(cat /var/opt/graphdb/node_dns)
-
+PROTOCOL=$(echo "${external_address_http_string_type}" | awk '{print tolower($0)}')
```

Comment:
This `tolower` can be done with Terraform when providing the variable to the user scripts.